pkg/annotation/machinedeployment.go

```diff
@@ -0,0 +1,6 @@
+package annotation
+
+// MachineDeploymentSubnet is used to specify the IPv4 CIDR used for a
+// node pool defined by a MachineDeployment or AWSMachineDeployment
+// resource.
```

Comment:
I think we should stick to established terminology and not derive from what people are used to. So either name it `CR` or `custom resource`. 

```suggestion
// node pool defined by a MachineDeployment or AWSMachineDeployment CR.
```